### ISSUE TITLE
Hide service's wsdl

### ISSUE THREAD

**Issue by 8lueFox** [DigDes/SoapCore#898]
I have simple structure in my web application. 
![image](https://user-images.githubusercontent.com/60403200/189355043-2c317a4b-26b5-4f44-af11-43b7b51ba6b0.png)

Is there any chance to hide a wsdl source in url https://localhost:7189/Service.svc to keep it unvisible from users?
![image](https://user-images.githubusercontent.com/60403200/189355209-9cbbad40-57d6-4226-8ba0-ec1faf99ce32.png)

**Comment by andersjonsson** [comment]
You don't want to show the wsdl at all?
There is an option to use a file as your wsdl. I haven't tried it personally, but I imagine that you could use an empty file.
https://github.com/DigDes/SoapCore#using-with-external-wsdl--xsd-schemas

**Comment by 8lueFox** [comment]
I was want to hide wsdl file from users in browser. I found information that I can block access to GET method and stay POST available. I made simple extension method which making services with denied GET. 

public static void UseSoapServiceWithDeniedGET<Service>(this IEndpointRouteBuilder endpoints)
         where Service : class
    {
        endpoints.UseSoapEndpoint<Service>(opt =>
        {
opt.Path = $"/{typeof(Service).Name}.svc";
opt.HttpGetEnabled = false;
opt.HttpsGetEnabled = false;
opt.EncoderOptions = new SoapEncoderOptions[] { new SoapEncoderOptions() };
            opt.SoapSerializer = SoapSerializer.XmlSerializer;
        });
    }

and then instead using useSoapEndpoint and giving manually all options I just call this extension method. 

`app.UseEndpoints(endpoints =>
{
    endpoints.UseSoapServiceWithDeniedGET<BookService>();
});`
